backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/DateUnionAggregator.java

```diff
@@ -7,13 +7,14 @@
 import com.bakdata.conquery.models.externalservice.ResultType;
 import com.bakdata.conquery.models.identifiable.ids.specific.TableId;
 import com.bakdata.conquery.models.query.QueryExecutionContext;
+import com.bakdata.conquery.models.query.queryplan.aggregators.Aggregator;
 import com.bakdata.conquery.models.query.queryplan.aggregators.SingleColumnAggregator;
 import com.bakdata.conquery.models.query.queryplan.clone.CloneContext;
 
 /**
  * Aggregator, listing all days present.
  */
-public class DateUnionAggregator extends SingleColumnAggregator<String> {
+public class DateUnionAggregator extends SingleColumnAggregator<Object[]> {
```

Comment:
```suggestion
public class DateUnionAggregator extends SingleColumnAggregator<CDateRange[]> {
```

Reply:
Ich würde es bei `Object[]` belassen, denn auf Manager-Seite kann nur ein  `Object[]` rauskommen

Reply:
Es macht im Code aber schon einen Unterschied, dadurch können wir sehr gut komponenten typsicher ineineinander hängen. Da das ja bei der Serialisierung verschwindet finde ich das zumindest in der QueryEngine noch gut das tpyisiert zu lassen

Reply:
Ich gebe jetzt eine `Collection<CDateRange>` raus. Dann müssen nicht extra Arrays aus dem `asRanges()` erstellt werden.